components/collection/CollectionCard.vue

```diff
@@ -55,17 +55,21 @@ const props = defineProps<{
 
 const isLoadingMeta = ref(false)
 const image = ref('')
+const banner = ref('')
 
 const { urlPrefix } = usePrefix()
 
 const getImageFromMetadata = async (collectionMetadata: string) => {
   isLoadingMeta.value = true
 
-  const metadata = (await processSingleMetadata(
-    collectionMetadata,
-  )) as TokenMetadata
+  const metadata = await processSingleMetadata<TokenMetadata>(collectionMetadata)
```

Comment:
update types? or using another types

Reply:
the type did not change just using generic types

Reply:
`metadata.banner` does not exists in this type, no?

Reply:
oh, yes this type doesn't have banner, should we update the `hyperdata`  type or create a local one